ui/admin/app/components/form/host/aws/index.hbs

```diff
@@ -11,51 +11,70 @@
   as |form|
 >
 
-  <form.input
+  <Hds::Form::TextInput::Field
     @value={{@model.type}}
-    @label={{t 'form.type.label'}}
-    readonly={{true}}
-    @disabled={{true}}
-  />
+    name='type'
+    disabled={{true}}
+    as |F|
+  >
+    <F.Label>{{t 'form.type.label'}}</F.Label>
+  </Hds::Form::TextInput::Field>
+
+  {{#if @model.external_name}}
+    <Hds::Form::TextInput::Field
+      @value={{@model.external_name}}
```

Comment:
should we add `isRequired/isOptional` arg here?

Reply:
I didn't add them to these fields because these are "non-standard" forms since they are basically readonly and can't be modified as the dynamic hosts are being pulled externally.

It didn't make sense to me to include `isOptional` or `isRequired` for these fields, any thoughts to this?

Reply:
Yeah, in that case, we don't need to.